### PR TITLE
Argbash

### PR DESCRIPTION
**Pull request by phozzy**

### CODE REVIEW

File: connector.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+version: '3'
+
+services:
+  transmart-variant-store-connector:
+    container_name: transmart-variant-store-connector
+    image: thehyve/transmart-variant-store-connector
```
Comment (gijskant) [inline connector.yml]: Maybe add version `:0.0.1`?

File: README.md [hunk 2]
```diff
@@ -211,6 +211,24 @@ The services can be stopped with `./stopall`.
     ./startall
     ```
 
+## Running variant store connector
+
+The `startall` script has a flag `--connector` that allows you to run your setup together with variant store connector.
+If you run this script without any flag or with `--no-connector`, your setup will run without variant store connector.
+
+Before running variant store connector be sure that your `.env` file contains all variables declaration as described preivously.
```
Comment (gijskant) [inline README.md]: preivously -> previously

File: startall [hunk 3]
```diff
@@ -1,3 +1,60 @@
...
+	test "$all_short_options" = "${all_short_options/$first_option/}" && return 1 || return 0
+}
+
+# THE DEFAULTS INITIALIZATION - OPTIONALS
+_arg_connector="off"
+
+
+print_help()
+{
+	printf '%s\n' "<The general help message of my script>"
```
Comment (gijskant) [inline startall]: Perhaps `Start all Glowing Bear and TranSMART components using docker-compose`?

File: stopall [hunk 4]
```diff
@@ -1,3 +1,60 @@
...
+	test "$all_short_options" = "${all_short_options/$first_option/}" && return 1 || return 0
+}
+
+# THE DEFAULTS INITIALIZATION - OPTIONALS
+_arg_connector="off"
+
+
+print_help()
+{
+	printf '%s\n' "<The general help message of my script>"
```
Comment (gijskant) [inline stopall]: Perhaps `Stop all Glowing Bear and TranSMART components using docker-compose`?

File: connector.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    ports:
+    - 9060:9060
+    environment:
+      KEYCLOAK_SERVER_URL: ${KEYCLOAK_SERVER_URL:?Please specify a Keycloak url.}
+      KEYCLOAK_REALM: ${KEYCLOAK_REALM:?Please specify a Keycloak realm.}
+      KEYCLOAK_CLIENT_ID: ${KEYCLOAK_CLIENT_ID:?Please specify a Keycloak resource.}
+      TRANSMART_API_SERVER_URL: ${TRANSMART_API_SERVER_URL:?Please specify a Transmart url.}
```
Comment (gijskant) [inline connector.yml]: This should be the internal address `http://transmart-api-server:8081` (see, e.g., `gb-backend`)

File: connector.yml (hunk 1 above)
Comment (phozzy) [reply connector.yml]: Ok. I'll do that. I was not sure if it is required.

File: startall (hunk 3 above)
Comment (phozzy) [reply startall]: :-D nice catch

File: README.md [hunk 6]
```diff
@@ -211,6 +211,24 @@ The services can be stopped with `./stopall`.
...
 
+## Running variant store connector
+
+The `startall` script has a flag `--connector` that allows you to run your setup together with variant store connector.
+If you run this script without any flag or with `--no-connector`, your setup will run without variant store connector.
+
+Before running variant store connector be sure that your `.env` file contains all variables declaration as described previously.
+You should also add additional variable to that file, so variant store connector will be able to find variant store instance and transmart server instance:
+```properties
+TRANSMART_API_SERVER_URL=https://transmart-dev.thehyve.net
```
Comment (phozzy) [inline README.md]: @gijskant shall I remove it, since this value is defined inside a compose file?

**Review by gijskant — APPROVED** [review]

**Pull request merged**